### PR TITLE
Early support of the Saigo toolchain

### PR DESCRIPTION
**Pull request by illwieckz**
See:

- https://github.com/DaemonEngine/Daemon/pull/981

### CODE REVIEW

**Comment by illwieckz** [comment]
First commit `cmake: glm and tinygettext are cgame/sgame things` was remade and merged as 6cff0a64975c01d832f4f2c057f0f38cdfeba15a

So only one commit is left.

File: CMakeLists.txt [hunk 1]
```diff
@@ -232,10 +232,12 @@ if (BUILD_SGAME)
 
     if (BUILD_GAME_NACL AND NOT (FORK EQUAL 2))
         include(ExternalProject)
-        ExternalProject_Add_Step(nacl-vms cbse-trigger
-                                 COMMENT "Finished CBSE generation, ready to build NaCl"
-                                 DEPENDS ${sgame_GENERATED_CBSE}
-                                 DEPENDERS build)
+        foreach(NACL_VMS_PROJECT ${NACL_VMS_PROJECTS})
+            ExternalProject_Add_Step(${NACL_VMS_PROJECT} cbse-trigger
+                COMMENT "Finished CBSE generation, ready to build NaCl"
```
Comment (slipher) [inline CMakeLists.txt:237]: How about adding the project name in the comment so the message isn't identical every time?

Comment (illwieckz) [reply CMakeLists.txt]: Done.

**Comment by slipher** [comment]
LGTM

**Pull request merged**